inc/patches/3-9-3-1.patch

```diff
@@ -111,3 +111,17 @@ index 976d4eb..155c1f6 100644
  #else
  /******************************************************************************/
  /**
+
+diff --git a/crypto/keyformat/pkcs.c b/crypto/keyformat/pkcs.c
+index f1a725e..afb537f 100644
+--- a/crypto/keyformat/pkcs.c
++++ b/crypto/keyformat/pkcs.c
+@@ -519,7 +519,8 @@ int32 psPkcs8ParsePrivBin(psPool_t *pool, unsigned char *buf, int32 size,
+         if (plen > 0)
+         {
+             /* Unexpected extra data remains. Treat it as an error. */
+-            goto PKCS8_FAIL;
++            psTraceIntCrypto("Unexpected extra data: %d\n", plen);
++            //goto PKCS8_FAIL;
```

Comment:
I did some digging and I'm not sure why they consider it an error. There is some extra data in our PKCS12 tests (6 bytes, looks like an OID) - needs further investigation why it's there and what it means.

Leave it as is, I will try to figure all out until the next update.